### ISSUE TITLE
Cylinder rendering error

### ISSUE THREAD

**Issue by pdelia** [pdelia/away3d#203]
#23 Issue by __GoogleCodeExporter__, created on: 2015-04-24T07:51:10Z

```
What steps will reproduce the problem?
1. Create a cylinder with segmentsH:2
2. Render view
3. Get error:
Error: isNaN(sz)
	at away3d.cameras::Camera3D/project()[/Users/jensa/Documents/Flex Builder
3/Away3D_HEAD/src/away3d/cameras/Camera3D.as:282]

What is the expected output? What do you see instead?
A rendered cylinder with two segments

This only happens if segmentsH == 2

There is also another and possibly related bug - increasing the number of
segmentsH also multiplies the height (if height is set to 50 and segmentsH
is set to 4, the rendered height will be 200).
```

Original issue reported on code.google.com by `jens...@gmail.com` on 28 Nov 2008 at 2:55

**Comment by pdelia** [comment]
Comment by __GoogleCodeExporter__, created on: 2015-04-24T07:51:11Z

```
do not see the same effect in the current trunk - is it possible to include a 
sample
code snippet to test?

Rob
```

Original comment by `rob.bate...@gmail.com` on 11 Dec 2008 at 7:54

**Comment by pdelia** [comment]
Comment by __GoogleCodeExporter__, created on: 2015-04-24T07:51:11Z

```
cone1 = new 
Cylinder({material:"red#black",segmentsH:1,segmentsW:15,heigth:70,radius:30,x:-1
70});
view.scene.addChild(cone1);
cone2 = new 
Cylinder({material:"orange#black",segmentsH:4,segmentsW:15,heigth:70,radius:30,x
:0,openEnded:true,bothsi
des:true});
view.scene.addChild(cone2);

In this example, cone2 will have four times the length of cone1



```

Original comment by `jens...@gmail.com` on 13 Dec 2008 at 1:46

**Comment by pdelia** [comment]
Comment by __GoogleCodeExporter__, created on: 2015-04-24T07:51:11Z

```
fixed and updated in trunk version 2.2.7
```

Original comment by `rob.bate...@gmail.com` on 5 Feb 2009 at 3:43
* Changed state: **Fixed**
